PocketKit/Sources/PocketKit/MyList/EmptyStates/EmptyStateView.swift

```diff
@@ -0,0 +1,90 @@
+import SwiftUI
+import Textile
+
+open class SwiftUICollectionViewCell<Content>: UICollectionViewCell where Content: View {
+    
+    private(set) var host: UIHostingController<Content>?
+    
+    func embed(in parent: UIViewController, withView content: Content) {
+        if let host = self.host {
+            host.rootView = content
+            host.view.layoutIfNeeded()
+        } else {
+            let host = UIHostingController(rootView: content)
+            parent.addChild(host)
+            host.didMove(toParent: parent)
+            self.contentView.addSubview(host.view)
+            self.host = host
+        }
+    }
+    
+    deinit {
+        host?.willMove(toParent: nil)
+        host?.view.removeFromSuperview()
+        host?.removeFromParent()
+        host = nil
+    }
```

Comment:
referenced this article - https://medium.com/expedia-group-tech/swiftui-with-uicollectionview-aba7cbaf6d16